croud/clusters/commands.py

```diff
@@ -67,8 +72,16 @@ def clusters_deploy(args: Namespace) -> None:
     if args.unit:
         body["product_unit"] = args.unit
     client = Client.from_args(args)
-    client.send(RequestMethod.POST, "/api/v2/clusters/", body=body)
-    client.print(keys=["id", "name", "fqdn", "url"])
+    data, errors = client.send(RequestMethod.POST, "/api/v2/clusters/", body=body)
+    print_response(
+        data=data,
+        errors=errors,
+        keys=["id", "name", "fqdn", "url"],
+        success_message=(
+            "Cluster deployed. It may take a few minutes to " "complete the changes."
```

Comment:
Similar changes in several other code parts

```suggestion
            "Cluster deployed. It may take a few minutes to complete the changes."
```